documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -2,6 +2,13 @@
 
 The Raspberry Pi Touch Display is an LCD display which connects to the Raspberry Pi through the DSI connector. In some situations, it allows for the use of both the HDMI and LCD displays at the same time (this requires software support).
 
+[.whitepaper, title="Implementing a DSI Driver on the Raspberry Pi", subtitle="", link=https://pip.raspberrypi.com/categories/685-whitepapers-app-notes/documents/RP-003472-WP/Using-a-DSI-display.pdf]
+****
+This whitepaper is an introduction to writing a Display Serial Interface (DSI) driver for liquid crystal display (LCD) panels running under the Kernel Mode Setting (KMS) graphics system. It does not cover DSI drivers for the legacy graphics system, or Fake Kernel Mode Setting (FKMS).
+
+It assumes that the Raspberry Pi is running the Raspberry Pi operating system (OS; Linux), and are fully up to date with the latest firmware and kernels.
```

Comment:
Can probably shorten "the Raspberry Pi operating system (OS; Linux)" to just "Raspberry Pi OS". And "are fully" -> "is fully".